utils/scripts/terraform/openstack/requirements.sh

```diff
@@ -0,0 +1,7 @@
+sudo dnf install -y https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
```

Comment:
Is this going to run ods-ci tests? Do we also need to pull down the chromedriver here?

Reply:
Yes, the main [Readme.md](https://github.com/red-hat-data-services/ods-ci/blob/master/README.md) file says that chrome and chromedriver should have the same version number. In the [Dockerimage](https://github.com/red-hat-data-services/ods-ci/blob/master/build/Dockerfile) there is an example about how to install required packages

Reply:
Have installed chromedriver as mentioned in Dockerfile. We need this step in case if we want to use the vm as static worker node.